charts/vela-workflow/crds/core.oam.dev_workflows.yaml

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
please make sure only one of them exists: 

* https://github.com/kubevela/workflow/tree/main/config/crd/bases
* charts/vela-workflow/crds/